main.cpp

```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bicycle
+//  action 1: the bicycle rides
+bicycle.ride();
+//  action 2: the bicycle flies
+bicycle.fly();
+//  action 3: the bicycle dings its bell
+bicycle.bells();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: lavalamp
+//  action 1: lavalamp is turned ON
+lavalamp.on();
+//  action 2: lavalamp rotates
+lavalamp.rotate();
 
-//  action 3:
+//  action 3: lavalamp plays music
+lavalamp.music();
 
 //  3)
-//  Noun:
-//  action 1:
+//  Noun: Oven
+//  action 1: Oven needs cleaning
+oven.needsCleanin();
 
-//  action 2:
+//  action 2: Oven makes buns
+oven.makesBuns();
 
-//  action 3:
+//  action 3: Oven light is broken
+oven.lightBroken();
 
 //  4)
-//  Noun:
-//  action 1:
+//  Noun: Ganesha
+//  action 1: Ganesha needs an offering
+ganesha.needsOffering();
 
-//  action 2:
+//  action 2: Ganesha grants you infinte awesome because you offered
+ganesha.grantsAwesome();
 
-//  action 3:
+//  action 3: Ganesha rubs belly
+ganesha.rubsBelly();
 
 //  5)
-//  Noun:
-//  action 1:
+//  Noun: Plants
+//  action 1: Plants needs watering
+plants.needsWatering();
 
-//  action 2:
+//  action 2: Plants photosynthesize
+plants.photosynthesis();
```

Comment:
`plants.photosynthesize();`
'synthesize' is the action.  'synthesis' is the noun.
your pseudocode needs to describe an action.
- [ ] revise